src/serverdlg.h

```diff
@@ -36,68 +36,72 @@
 #include "settings.h"
 #include "ui_serverdlgbase.h"
 
-
 /* Definitions ****************************************************************/
 // update time for GUI controls
-#define GUI_CONTRL_UPDATE_TIME      1000 // ms
+#define GUI_CONTRL_UPDATE_TIME 1000 // ms
 
 // Strings used in multiple places
 #define SREC_NOT_INITIALISED CServerDlg::tr ( "Not initialised" )
-#define SREC_NOT_ENABLED     CServerDlg::tr ( "Not enabled" )
-#define SREC_NOT_RECORDING   CServerDlg::tr ( "Not recording" )
-#define SREC_RECORDING       CServerDlg::tr ( "Recording" )
-
+#define SREC_NOT_ENABLED CServerDlg::tr ( "Not enabled" )
+#define SREC_NOT_RECORDING CServerDlg::tr ( "Not recording" )
+#define SREC_RECORDING CServerDlg::tr ( "Recording" )
 
 /* Classes ********************************************************************/
 class CServerDlg : public CBaseDlg, private Ui_CServerDlgBase
 {
     Q_OBJECT
 
-public:
+  public:
     CServerDlg ( CServer*         pNServP,
                  CServerSettings* pNSetP,
                  const bool       bStartMinimized,
                  QWidget*         parent = nullptr );
 
-protected:
+  protected:
     virtual void changeEvent ( QEvent* pEvent );
     virtual void closeEvent ( QCloseEvent* Event );
 
-    void         UpdateGUIDependencies();
-    void         UpdateSystemTrayIcon ( const bool bIsActive );
-    void         ShowWindowInForeground() { showNormal(); raise(); }
-    void         ModifyAutoStartEntry ( const bool bDoAutoStart );
-    void         UpdateRecorderStatus( QString sessionDir );
+    void UpdateGUIDependencies();
+    void UpdateSystemTrayIcon ( const bool bIsActive );
+    void ShowWindowInForeground()
+    {
+        showNormal();
+        raise();
+    }
+    void ModifyAutoStartEntry ( const bool bDoAutoStart );
+    void UpdateRecorderStatus ( QString sessionDir );
 
-    QTimer                    Timer;
-    CServer*                  pServer;
-    CServerSettings*          pSettings;
+    QTimer           Timer;
+    CServer*         pServer;
+    CServerSettings* pSettings;
 
     CVector<QTreeWidgetItem*> vecpListViewItems;
     QMutex                    ListViewMutex;
 
-    QMenuBar*                 pMenu;
+    QMenuBar* pMenu;
 
-    bool                      bSystemTrayIconAvaialbe;
-    QSystemTrayIcon           SystemTrayIcon;
-    QPixmap                   BitmapSystemTrayInactive;
-    QPixmap                   BitmapSystemTrayActive;
-    QMenu*                    pSystemTrayIconMenu;
+    bool            bSystemTrayIconAvaialbe;
+    QSystemTrayIcon SystemTrayIcon;
+    QPixmap         BitmapSystemTrayInactive;
+    QPixmap         BitmapSystemTrayActive;
+    QMenu*          pSystemTrayIconMenu;
 
-public slots:
+  public slots:
     void OnRegisterServerStateChanged ( int value );
     void OnStartOnOSStartStateChanged ( int value );
     void OnEnableRecorderStateChanged ( int value )
-        { pServer->SetEnableRecording ( Qt::CheckState::Checked == value ); }
+    {
```

Comment:
But, as said, not these single statement ones.  All on one line, or wrap the body, keeping it on one line.